bib/american.bib

```diff
@@ -3348,7 +3348,7 @@ @BOOK(
 	year = 1846,
 	url = {http://hdl.loc.gov/loc.music/sm1846.060960},
 	keywords = {music | dance description | dance description: short | polka | quadrille | quadrille: london polka quadrille},
-	filename = {/Short descriptions/19th century/Stephen Glover/[1846, us] Stephen Glover - The London Polka Quadrilles (Lee & Walker copy).pdf},
+	filename = {/Short descriptions/19th century/Stephen Glover/[1846, us] Stephen Glover - The London Polka Quadrilles (Lee {\&} Walker copy).pdf},
```

Comment:
This is not required, since filename field isn't used by latex compilation routine